modules/ROOT/pages/consuming-a-soap-api.adoc

```diff
@@ -0,0 +1,17 @@
+= Consuming a SOAP API
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: anypoint, studio, connectors, soap, wsdl, api
+
+If you want to consume an API that does not already have its own Anypoint Connector, use the *Web Service Consumer* connector to consume any SOAP API from within your Mule application.
+
+Using the information contained in the API's WSDL, this connector enables you to configure a few details to establish the connection you need for consuming a service from within your Mule application. +
```

Comment:
"configure a few details to establish the connection" -> "provide a connection configuration" or something similar.

I'm wondering if we really need this page in the Mule Runtime docs. We're just pointing people out to the WSC connector doc.

Reply:
Yes, I thought the same originally and thus confirmed with the devs that they want this in another doc, because the Publishing SOAP APIs is too long to have this included and prefer to be separated.
- Consuming SOAP APIs
- Publishing SOAP APIs

I'll have all these docs prepared for migration but label them as DO NOT MERGE YET, so we can migrate them together.